.travis.yml

```diff
@@ -11,13 +11,18 @@ services:
 
 install: scripts/travis-setup.sh
 
-script: travis_wait 119 scripts/travis-run.sh
+script: scripts/travis-run.sh
 
 env:
   global:
     - SUBDAGS=1
     - BIOCONDA_UTILS_TAG=master
```

Comment:
The [latest](https://github.com/bioconda/bioconda-utils/commit/415709197fc3390aecbb4ab61e90950b5a192da5)?